forem/forem
# Remove chars-count from user fields; fix inconsistent loading

## What type of PR is this? (check all applicable)


## Description
On "Settings" tab load, the character counts do not load consistently. They only load consistently on page reload. This PR attempts to fix that. I also remove the character counts from Name, Username and Email fields (as recommended by Design/Product).

## Related Tickets & Documents
None

## QA Instructions, Screenshots, Recordings
Navigate to `localhost:3000` and from the User Profile dropdown, choose `Settings`. Ensure the character counts for the form fields load correctly, and are updated when the field is updated.

### UI accessibility concerns?
Taken care of.

## Added/updated tests?

- [X] No, and this is why: _Unnecessary_

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._
